siriuspy/siriuspy/devices/fofb.py

```diff
@@ -1071,6 +1072,35 @@ def cmd_fofbacc_clear(self, psnames=None, psindices=None):
             dev.cmd_fofbacc_clear()
         return True
 
+    def set_fofbacc_filter(self, values, psnames=None, psindices=None):
+        """Command to set power supply filter coefficients values."""
+        if not isinstance(values, (list, tuple, _np.ndarray)):
+            raise ValueError('Value must be iterable.')
+        devs = self._get_devices(psnames, psindices)
+        if not len(values) == len(devs):
+            raise ValueError('Values and indices must have the same size.')
+        for i, dev in enumerate(devs):
+            dev.fofbacc_filter = values[i]
+        return True
+
+    def check_fofbacc_filter(
+            self, values, psnames=None, psindices=None,
+            atol=DEF_ATOL_ACCFILTER):
+        """Check power supplies filter coefficients."""
+        if not self.connected:
+            return False
+        if not isinstance(values, (list, tuple, _np.ndarray)):
+            raise ValueError('Value must be iterable.')
+        devs = self._get_devices(psnames, psindices)
+        if not len(values) == len(devs):
+            raise ValueError('Values and indices must have the same size.')
+        for i, dev in enumerate(devs):    
+            if len(values[i]) != len(dev.fofbacc_filter):
+                return False
```

Comment:
Due to some hardware problems, I think it would be best to implement the low level so it returns a 20 element array only. So this shouldn't be a problem for you anymore.